components/ui/DataTable/Columns.tsx

```diff
@@ -0,0 +1,222 @@
+'use client';
+
+import { type ColumnDef } from '@tanstack/react-table';
+import { type Interview, type Protocol, type User } from '@prisma/client';
```

Comment:
Where everything is a type, you can just put type before the named imports:

```suggestion
import type { Interview, Protocol, User } from '@prisma/client';
```